workspaces/tech-insights/.changeset/add-metadata.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage-community/plugin-tech-insights-backend-module-jsonfc': patch
+'@backstage-community/plugin-tech-insights-common': patch
+'@backstage-community/plugin-tech-insights-node': patch
```

Comment:
node package already follows semver best practices and is above 1.x so let's follow the conventions and make this a minor change.

Reply:
Thanks @Xantier, updated description to minor in new commit